extension_helpers/conftest.py

```diff
@@ -27,9 +30,6 @@ def pytest_unconfigure(config):
 
     if HAS_COVERAGE:
```

Comment:
Do we expect the coverage checks to monitor these lines or not? If not should we exclude them? It's not clear if they aren't working or if it's just not recording them.

Reply:
ISTM none of `pytest_unconfigure` is covered, possibly because it runs after the pytest coverage plugin has done its work.  I think we can just exclude it from the coverage report.  Do you know how to do that?

Reply:
Pragma: nocover?